css/properties/font-family.json

```diff
@@ -74,15 +74,25 @@
               "oculus": "mirror",
               "opera": "mirror",
               "opera_android": "mirror",
-              "safari": {
-                "alternative_name": "-apple-system",
-                "version_added": "9",
-                "notes": "Supported since macOS 10.11."
-              },
-              "safari_ios": {
-                "alternative_name": "-apple-system",
-                "version_added": "9"
-              },
+              "safari": [
+                {
+                  "version_added": "11"
+                },
+                {
+                  "alternative_name": "-apple-system",
+                  "version_added": "9",
+                  "notes": "Supported since macOS 10.11."
```

Comment:
I don't think we should have notes like these. It's often the case that a feature depends on a new macOS release, but we don't say so in BCD.

Per https://en.wikipedia.org/wiki/Safari_version_history#Safari_9 Safari 9 was released at the same time as macOS 10.11, but also supported on 10.9 and 10.10.

Reply:
Correct, this would have been supported on all OS releases that Safari 9 shipped with. Only certain rare cases does a feature lock with the OS version (usually image/video encoding formats).

Reply:
Thanks for confirming, @xeenon!